symengine/python/symengine/lib/symengine_wrapper.pyx

```diff
@@ -1698,9 +1745,374 @@ def powermod_list(a, b, m):
         s.append(c2py(<RCP[const symengine.Basic]>(v[i])))
     return s
 
+# Is this really intentional? eval_double already defined earlier:
```

Comment:
As you see this is a redifinition of eval_double. Should I remove it (or the other)?